pkg/steps/artifacts.go

```diff
@@ -181,16 +190,14 @@ func copyArtifacts(podClient PodClient, into, ns, name, containerName string, pa
 		args = append(args, "-C", s, ".")
 	}
 
-	u := podClient.RESTClient().Post().Resource("pods").Namespace(ns).Name(name).SubResource("exec").VersionedParams(&coreapi.PodExecOptions{
+	e, err := podClient.Exec(ns, name, &coreapi.PodExecOptions{
```

Comment:
nit: the number of single letter vars here makes this hard to read

Reply:
I tried to touch the existing code as little as possible, mostly moved things around.